docs/source/getting-started/index.rst

```diff
@@ -66,7 +66,7 @@ In order to connect the UI code to this DAML, we need to run a code generation s
 Now, changing to the ``ui`` folder, use Yarn to install the project dependencies::
 
     cd ui
-    yarn install
+    yarn install --force --frozen-lockfile
```

Comment:
I see where you’re coming from but I’m not particularly keen on adding this rather gross hack in even more places than necessary. Was this a frequent issue?

Reply:
Unfortunately yes, users encountered it all the time when they were trying to add new code. They would really forget that whenever you need to make the changes in the UI you need to call `yarn install` with the extra parameters. This lead to endless loops of checking what's wrong in all the wrong places 😢 and time waste. 

For me this is really a temporary fix until we come up with a nicer build process (@hurryabit said that someone form the team will be working on it).

Reply:
Alright then let’s include that for now.